tools/Jenkinsfile-PR

```diff
@@ -0,0 +1,24 @@
+pipeline {
+	environment {
+		ANDROID_HOME = '/opt/imint/android-sdk'
+		ANDROID_NDK_HOME = '/home/jenkins/android-ndk-r15c'
+		DISPLAY = ':0'
+	}
```

Comment:
Does the kean tests need a display?
If not you can remove the whole `environment` part, otherwise you can remove the android stuff.

Reply:
A display is needed. As for android, I may be incorrect, but isn't the NDK needed to build?

Reply:
I don't think kean need android to build? Also if it did it wouldn't look for those two environment variables I think.